src/nodes.coffee

```diff
@@ -1158,24 +1165,45 @@ exports.Obj = class Obj extends Base
         prop.variable
       else if prop not instanceof Comment
         prop
-
-      if key instanceof Value and key.hasProperties()
+      # Pass the splat through.
+      if key instanceof Value and key.hasProperties() and key not instanceof Splat
```

Comment:
Yep, I agree. I think I added this before your PR was merged when I tried to fix some errors.